src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -61,10 +67,29 @@ const UserProfileModal: React.FunctionComponent<PropTypes> = ({
     return undefined;
   }, [rest]);
 
+  const chatContext = useContext(ChatContext);
+
+  const submitMessage = useCallback(
+    async (data: { messageToTheBand: string }) => {
+      chatContext &&
+        user &&
+        chatContext.sendPrivateChat(
+          user.uid,
+          fullUserProfile!.id,
```

Comment:
Do we want to be using `!` here?

Reply:
Also, what does `fullUserProfile` refer to here?

`chatContext.sendPrivateChat` seems to take `(from, to, text)` as params, implying we are sending a message to `fullUserProfile.id` here?

I'm guessing that since this is passed down from props, the implication is that `fullUserProfile` is the user that this `UserProfileModal` is for/showing. And `user` is the current person that opened the modal.

The ambiguous use of `user` in this component makes it hard to follow. I think renaming `user` to something implying "the current person" (`loggedInUser`, `authenticatedUser`, `myUser`?), and/or renaming `fullUserProfile` to something implying "the person this modal is about" would make this a lot easier to follow.